tests/db_utils/db_setup.py

```diff
@@ -132,49 +144,47 @@ def get_insertion_order(conn, gt_csv_files):
 
   gt_dir = f'{data_dir}/ground_truth'
   gt_csv_fnames = glob.glob(f'{gt_dir}/*.csv')
-  # Define an asynchronous function to insert data
-  async with engine.begin() as conn:
-    gt_csv_fnames = await conn.run_sync(get_insertion_order, gt_csv_fnames, )
-    # Create tables
-    for csv_fname in gt_csv_fnames:
-      base_fname = os.path.basename(csv_fname)
-      table_name = base_fname.split('.')[0]
-
-      if only_blob_data and not table_name.startswith('blobs') and not table_name.startswith('mapping_'):
-        continue
-      df = pd.read_csv(csv_fname)
-      for column in df.columns:
-        column_info = extract_column_info(table_name, column)
-        df.rename(columns={column: column_info.name}, inplace=True)
 
-      if table_name.startswith('mapping'):
-        _, _, blob_mapping_table_name = table_name_for_rep_and_topk_and_blob_mapping(['blobs_00'])
-        table_name = blob_mapping_table_name
-      # FIXME: in case of mysql, this function doesn't wait, hence throwing integrity error
-      await conn.run_sync(lambda conn: df.to_sql(table_name, conn, if_exists='append', index=False))
+  # async with engine.begin() as conn:
+  gt_csv_fnames = await conn.run_sync(get_insertion_order, gt_csv_fnames, )
+  # Create tables
+  for csv_fname in gt_csv_fnames:
+    base_fname = os.path.basename(csv_fname)
+    table_name = base_fname.split('.')[0]
+
+    if only_blob_data and not table_name.startswith('blobs') and not table_name.startswith('mapping_'):
+      continue
+    df = pd.read_csv(csv_fname)
+    for column in df.columns:
+      column_info = extract_column_info(table_name, column)
+      df.rename(columns={column: column_info.name}, inplace=True)
+
+    if table_name.startswith('mapping'):
+      _, _, blob_mapping_table_name = table_name_for_rep_and_topk_and_blob_mapping(['blobs_00'])
+      table_name = blob_mapping_table_name
 
+    await conn.run_sync(lambda conn: df.to_sql(table_name, conn, if_exists='append', index=False))
 
-async def clear_all_tables(engine):
+
+async def clear_all_tables(conn):
   def tmp(conn):
     metadata = MetaData()
     metadata.reflect(conn)
     for table in metadata.sorted_tables:
       if table.name.startswith('blobs'):
         continue
       conn.execute(table.delete())
-
-  async with engine.begin() as conn:
-    await conn.run_sync(tmp)
+  await conn.run_sync(tmp)
 
 
-async def setup_config_tables(engine):
+async def setup_config_tables(conn):
   def create_blob_metadata_table(conn):
     Base = declarative_base()
 
     class BlobTables(Base):
       __tablename__ = BLOB_TABLE_NAMES_TABLE
-      table_name = sqlalchemy.Column(sqlalchemy.String, primary_key=True)
-      blob_key = sqlalchemy.Column(sqlalchemy.String, primary_key=True)
+      table_name = sqlalchemy.Column(sqlalchemy.String(30), primary_key=True)
```

Comment:
String length is required for MySQL database

Reply:
Make this longer - lots of automatically generated schemas have long names, like 1024

Reply:
For the primary key column, we can only set the maximum length up to 256